Update common-issues.md (#477)
codemagic-ci-cd/codemagic-docs: content/troubleshooting/common-issues.md

**Solution**:
* Confirm that you have set the correct bundle identifiers for all targets in Xcode.
* Verify that the bundle identifier set in Xcode matches the bundle identifier set in the Flutter workflow editor under code signing.

> ### Common Podfile issue

Reviewer: The title should be more descriptive about the issue, eg. iOS deployment target issue

> The following issue is usually occurs when it comes to podfile issues:

Reviewer: A common error message with regard to deployment target is the following:

```
The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99
```

> 

Reviewer: Add here:
The error usually occurs if the deployment target set in Podfile is lower than required or when the Podfile is missing.

**Solution**:

> * Confirm if Podfile exists in repo if not, then create Podfile in `ios` directory and insert the following manually in there:

Reviewer: ```suggestion
* Confirm if a Podfile exists in the repository. If not, create a Podfile in the `ios` directory and insert the following manually in there:
```

```
# Uncomment the next line to define a global platform for your project
platform :ios, ’10.0’
target 'Runner' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!
 # Pods for Runner
end
```

> * Check Podfile, if `platform :ios, ’10.0’` is set to `10.0` or above as required by dependencies 

Reviewer: ```suggestion
* Check the Podfile and make sure that `platform :ios, ’10.0’` is set to `10.0` or above as required by dependencies.
```
